tests/data/test_conf.yaml

```diff
@@ -47,4 +47,4 @@ LADDER_SEARCH_EXPANSION_MAX: 0.25
 LADDER_SEARCH_EXPANSION_STEP: 0.05
 QUEUE_POP_TIME_MAX: 180
 QUEUE_POP_DESIRED_MATCHES: 4
-QUEUE_POP_TIME_MOVING_AVG_SIZE: 5
+QUEUE_POP_TIME_MOVING_AVG_SIZE: 5
```

Comment:
No pls

Reply:
I am confused what changed here I will just change it back

Reply:
You deleted the trailing newline